scripts/deeplink-dispatcher.sh

```
@@ -0,0 +1,316 @@
+        echo "Invalid selection. Please try again."
+        handle_ios "$link"
+      fi
+    fi
+  fi
+}
+
+# Function to handle sending link to Android
+handle_android() {
+  local link="$1"
+  local android_emus
+  android_emus=($(detect_android_emulators))
```

Shellcheck

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)